.eslintrc.json

```diff
@@ -0,0 +1,134 @@
+{
+  "plugins": [
+    "eslint-plugin-import",
+    "eslint-plugin-prefer-arrow",
+    "@typescript-eslint",
+    "ban"
+  ],
+  "extends": ["prettier"],
+  "rules": {
+    "@typescript-eslint/member-ordering": [
+      "error",
+      {
+        "default": ["field", "constructor", "method"]
+      }
+    ],
+    "@typescript-eslint/naming-convention": [
+      "error",
+      {
+        "selector": "variable",
+        "format": [
+          "camelCase", "UPPER_CASE", "PascalCase"
+        ]
+      },
+      {
+        "selector": "class",
+        "format": ["PascalCase"]
+      },
+      {
+        "selector": "interface",
+        "format": ["PascalCase"],
+        "custom": {
+          "regex": "^I[A-Z]",
+          "match": false
+        }
+      },
+      {
+        "selector": "property",
+        "format": ["camelCase", "UPPER_CASE", "PascalCase", "snake_case"],
+        "leadingUnderscore": "allow"
+      }
+    ],
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-inferrable-types": [
+      "error",
+      {
+        "ignoreParameters": true
+      }
+    ],
+    "@typescript-eslint/no-require-imports": "error",
+    "@typescript-eslint/no-unused-expressions": "error",
+    "@typescript-eslint/prefer-for-of": "error",
+    "ban/ban": [
+      "error",
+      { "name": "fdescribe", "message": "Banned method" },
+      { "name": "fit", "message": "Banned method" },
+      { "name": "test.only", "message": "Banned method" }
```

Comment:
we would also want: `it.only`, `describe.only`, `logOut`

Reply:
Added 👍